sdk/scheduler/src/main/java/com/mesosphere/sdk/state/StateStoreUtils.java

```diff
@@ -33,7 +39,7 @@
     private static final String UNINSTALLING_PROPERTY_KEY = "uninstalling";
     private static final String LAST_COMPLETED_UPDATE_TYPE_KEY = "last-completed-update-type";
     private static final String PROPERTY_TASK_INFO_SUFFIX = ":task-status";
-    private static final int MAX_VALUE_LENGTH_BYTES = 1024 * 1024; // 1MB
+    protected static final int MAX_VALUE_LENGTH_BYTES = 1024 * 1024; // 1MB
```

Comment:
Ah. Ok, I misunderstood. I have made it private again and added a getter which is better practice in general, right? Or would you be happier with `@VisibleForTesting protected`?